proto/session/v1/service_streaming.proto

```diff
@@ -18,7 +18,7 @@ syntax = "proto3";
 
 package oak.session.v1;
 
-import "oak_remote_attestation/proto/v1/messages.proto";
+import "proto/session/v1/messages.proto";
```

Comment:
I think we are no longer using `v1` in the directory names in the new `proto` direcroty. `v1` is only mentioned in the package names.

So the directory could just be `proto/session/file.proto`

Reply:
Removed the v1 subdir but left v1 in namespaces - please let me know if you'd prefer them to go too!